### PR TITLE
[TF FE] Document full list of TF operations and their support by TF FE

### PR DESCRIPTION
**Pull request by rkazants**
**Details:** Document full list of TF operations from [tf.raw_ops](https://www.tensorflow.org/api_docs/python/tf/raw_ops)

**Ticket:** TBD

### CODE REVIEW

**Review by gkrivor — APPROVED** [review]

File: src/frontends/tensorflow/docs/supported_ops.md
Comment (mvafin) [inline src/frontends/tensorflow/docs/supported_ops.md:1]: Why we add `docs` directory in frontend when we have list of operations in https://github.com/openvinotoolkit/openvino/blob/5dafee4ac16e1cf757a1ba38008bdf5db0ae0e69/docs/articles_en/about_openvino/compatibility_and_support/supported_operations_framework_frontend.md?plain=1#L749

Comment (rkazants) [reply src/frontends/tensorflow/docs/supported_ops.md:1]: This one is only for developers to understand the gap. The other one for users sort of.
May be in some future we will merge these two docs.

Comment (mvafin) [reply src/frontends/tensorflow/docs/supported_ops.md:1]: What is the difference? It looks in the other one we have outdated info

**Comment by mvafin** [comment]
I am not convinced this is really needed, but will not block the PR.

File: src/frontends/tensorflow/docs/supported_ops.md
Comment (rkazants) [reply src/frontends/tensorflow/docs/supported_ops.md:1]: The difference is that users docs contain only supported ops. This one is developer documentation that expresses the gap what we need to contribute more. This table gives us idea about next GFIs and what we can contribute.

**Comment by rkazants** [comment]
> I am not convinced this is really needed, but will not block the PR.

Let me 

> I am not convinced this is really needed, but will not block the PR.

Discussed offline concerns

File: src/frontends/tensorflow/docs/supported_ops.md [hunk 1]
```diff
@@ -0,0 +1,1406 @@
+# TensorFlow Operations Supported by OpenVINO TensorFlow Frontend 
+
+Here is a table of supported operations by TensorFlow Frontend from [tf.raw_ops](https://www.tensorflow.org/api_docs/python/tf/raw_ops).
+Supported operation means that TensorFlow Frontend is capable to convert it into OpenVINO representation.
```
Comment (kblaszczak-intel) [inline src/frontends/tensorflow/docs/supported_ops.md:4]:
```suggestion
Here is a table of operations supported by the TensorFlow Frontend from [tf.raw_ops](https://www.tensorflow.org/api_docs/python/tf/raw_ops).
A "supported operation" is one that TensorFlow Frontend can convert to the OpenVINO representation.
```

**Pull request merged**